NERDweb/nerd_main.py

```diff
@@ -1242,6 +1242,10 @@ def iplist():
         return Response('ERROR: Database connection error', 503, mimetype='text/plain')
 
 
+# ******************** Map ********************
+@app.route('/map/')
+def map():
```

Comment:
Here you overwrite the builtin `map` function, which breaks everything. Rename it.